lib/twitter-ads/resources/analytics.rb

```diff
@@ -173,11 +189,11 @@ def check_async_job_status(account, opts = {})
         # set default values
         job_ids = opts.fetch(:job_ids, nil)
         params = {}
-        params[:job_ids] = Array.wrap(job_ids).join(',') if job_ids
```

Comment:
_bug_: Array#wrap is only available for ActiveSupport in Rails. This was introduced in https://github.com/twitterdev/twitter-ruby-ads-sdk/commit/b1f2e9764d262447464f2341b2a2af483f7635a4#diff-697bfc02b2bb3faa338ddbb781e0fb93 (2017)